lib/matplotlib/backends/backend_agg.py

```diff
@@ -543,6 +549,11 @@ def print_jpg(self, filename_or_obj, *args, dryrun=False, **kwargs):
             progressive : bool
                 If present, indicates that this image
                 should be stored as a progressive JPEG file.
+
+            pil_kwargs : dict, optional
+                Additional keyword arguments that are passed to Pillow when
```

Comment:
~~~
... passed to ``PIL.Image.save()``
~~~
or
~~~
... passed to Pillow ``Image.save()``
~~~

Or, even better include Pillow in intersphinx and link to the save method. That way, people can easily look up the possible parameters.

Reply:
done

Reply:
Please add the reference `PIL.Image.save()` here as well.

Reply:
done twice